src/rust/engine/dep_inference/Cargo.toml

```diff
@@ -1,11 +1,17 @@
 [package]
+# NB:
```

Comment:
Were you wanting to write more text here?

Reply:
Curious. Legend says that only those true of heart and pure of character can read the sacred `NB`. Whatever happened in your past, you should make peace with the gods, so that you can also perceive the holiest of messages.

Reply:
```suggestion
```